Makefile

```diff
@@ -68,6 +68,10 @@ gotasks: format lint build
 
 .PHONY: test
 test:
+	@echo "------------------"
+	@echo "Aquire YAML for performing tests"
+	@echo "------------------"
+	@mkdir -p ./vendor/github.com/litmuschaos/chaos-operator/deploy;wget -q -N https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/chaos_crds.yaml -P ./vendor/github.com/litmuschaos/chaos-operator/deploy ;wget -q -N https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/rbac.yaml -P ./vendor/github.com/litmuschaos/chaos-operator/deploy; wget -q -N https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/operator.yaml -P ./vendor/github.com/litmuschaos/chaos-operator/deploy
```

Comment:
These is good but instead of changing in vendor dir. can you get those file in `build/_output/test`